src/main/java/seedu/address/model/InfoHandler.java

```diff
@@ -231,24 +250,38 @@ public VisitBook generateVisitsByPerson(Id personId) {
     }
 
     /**
-     * Generates a list of visits that are associated with the given list of location Ids.
-     * @param locationIds List of location Ids.
-     * @return List of visits that are associated with the location Ids.
+     * Generates a list of visits that are associated with the given visits.
+     * @param visitBook Visits provided.
+     * @return List of visits that are associated with the given visits.
      */
-    public VisitBook generateVisitsByLocationIds(List<Id> locationIds) {
+    public VisitBook generateAffectedVisits(VisitBook visitBook) {
```

Comment:
Similarly, what does a visit being affected mean? I don't quite understand this method

Reply:
Also included criteria and explanation that it is an intermediate method.